### DISCUSSION TITLE
Unable to retrieve more than 100 Tweets.

### DISCUSSION THREAD

**Discussion by parikshitkoya** [tweepy/tweepy#1481] [Questions] [answered]
Hi,

I am fairly new to Python and Tweepy and trying to extract 3000 tweets for a class sentiment analysis project where a particular handle has been mentioned.

Below is my code:

```
import tweepy
from secrets import apikey, apikeysecret, accesstoken, accesstokensecret #My API Credentials

auth = tweepy.OAuthHandler(apikey, apikeysecret)
auth.set_access_token(accesstoken, accesstokensecret)

api = tweepy.API(auth)

tweets = api.search(q='@amazonin', count = 3000)
tweets
```

However, I am only getting 100 tweets with this query despite providing a count of 3000. I am not sure if passing the same request multiple times using a for loop with some delay would help as I would get a time out request from the server due to API Rate Limits. Please advise.

**Comment by Harmon758** [comment, marked as answer]
The `count` parameter for the [standard search API](https://developer.twitter.com/en/docs/twitter-api/v1/tweets/search/api-reference/get-search-tweets) that `API.search` uses has a maximum of 100.
For pagination, you should use a [Cursor](https://tweepy.readthedocs.io/en/latest/cursor_tutorial.html).

**Reply by parikshitkoya** [reply]
Thanks @Harmon758.

This has worked like a charm. 👍
